tests/e2e-test.spec.js

```diff
@@ -43,17 +43,88 @@ describe('Main workflow Test Suite', () => {
   after(async () => await client?.end());
 
   it('should have data in postgres medic table', async () => {
-    const couchdbTableResult = await client.query(`SELECT * FROM ${POSTGRES_SCHEMA}.${POSTGRES_TABLE}`);
-    expect(couchdbTableResult.rows.length).to.equal(docs.length * dbNames.length);
+    const couchdbTableResult = await client.query(`SELECT * FROM ${PGTABLE}`);
+    expect(couchdbTableResult.rows.length).to.equal(docs.length);
   });
 
-  it('should have data in postgres person table', async () => {
-    const personTableResult = await client.query(`SELECT * FROM ${DBT_POSTGRES_SCHEMA}.person`);
-    expect(personTableResult.rows.length).to.equal(persons().length * dbNames.length);
+  it('should have data in postgres contacts table', async () => {
+    const contactsTableResult = await client.query(`SELECT * FROM ${pgSchema}.contacts`);
+    expect(contactsTableResult.rows.length).to.equal(contacts().length);
   });
 
   it('should have data in postgres data_record table', async () => {
-    const dataRecordTableResult = await client.query(`SELECT * FROM ${DBT_POSTGRES_SCHEMA}.data_record`);
-    expect(dataRecordTableResult.rows.length).to.equal(dataRecords().length * dbNames.length);
+    const reportsTableResult = await client.query(`SELECT * FROM ${pgSchema}.reports`);
+    expect(reportsTableResult.rows.length).to.equal(reports().length);
+  });
+
```

Comment:
Can we add a validation also for `persons` table? To test all the models defined for testing in`/tests/dbt/models`?